docs/reference/upgrade.asciidoc

```diff
@@ -47,12 +47,15 @@ a|
 
 [WARNING]
 ====
-The following upgrade paths are *not* supported:
+The following upgrade path is *not* supported (both full cluster restart and rolling upgrade):
```

Comment:
```suggestion
The following upgrade path is *not* supported:
```

Reply:
I would keep it because it makes more explicit we're not talking only about rolling upgrades (as most of the page is).